lib/classes/Variables.js

```diff
@@ -760,7 +760,7 @@ class Variables {
 
   buildOptions(regionSuffix) {
     const options = { useCache: true };
-    if (!_.isUndefined(regionSuffix)) {
+    if (regionSuffix !== undefined) {
```

Comment:
Let's replace it with `if (!regionSuffix)`, as in this context we rather check whether `regionSuffix` was passed, and we should discard any _falsy_ value

Reply:
I'll carefully check this. Thanks!